sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,72 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+    defNum = 1
+    defLast = False
+    hasPrinted = False
+    defURL = r"https://www.merriam-webster.com/dictionary/"
+
+    if not args:
+        await message.channel.send(cmd.help())
+        return
+    try:
+        defNum = int(args[-1])
+        defLast = True
+    except ValueError:
+        pass
+
+    if not defLast:
+        for i in args:
+            defURL += i
+            defURL += "%20"
+    else:
+        argsCopy = args.copy()
+        del(argsCopy[-1])
+        for i in argsCopy:
+            defURL += i
+            defURL += "%20"
+
```

Comment:
By default, the first definition is given, which is usually enough to satisfy the necessity to use the command. However, some words can have radically different meanings, and this is where the option to retrieve a different meaning comes in handy. If we were to just give the user a random definition, it would not be very useful, because they might receive the definition they were looking for; and if they don't, they could always try again with a different number.